docs/cpp/guides/flags.md

```diff
@@ -377,7 +387,9 @@ remove reference to the flag from configuration files and startup scripts. Once
 all jobs are starting up without logging warnings about reference to the retired
 flag, the retired flag can be removed completely.
 
-## Special Usage Flags
+For more details, see the [Tip of the Week on retired flags][retired-flags].
```

Comment:
Maybe just abseil.io/tips/90 instead of indirecting the link?

Reply:
I dunno. I prefer this rather than naked links